src/services/squidrouter/route.ts

```diff
@@ -152,3 +150,25 @@ export async function getRouteTransactionRequest(
     data: routeResult.data,
   };
 }
+
+export async function testRoute(testingToken: InputTokenDetails, attemptedAmountRaw: string) {
+  const { fromChainId, toChainId, axlUSDC_MOONBEAM } = squidRouterConfig;
+  const mockFromAddress = '0x2527db2c3dc99db4ab7689e1972bf7168bfe0417'; // realy shouldn't matter
```

Comment:
It's probably fine to work with the mock address, but shouldn't we have the user address available at this point (the wallet should be connected)? That would make it possibly more robust.

Reply:
Yes we have it since we use this function only after the user confirms. But I thought we could use this one later in a more standalone way, perhaps in a test or at app initiation. I'll add it though.